make_level.py

```diff
@@ -47,49 +40,31 @@ def write_init(skill_name, skill_path, level_name):
             print("Registered skill \"{}\" in {}".format(skill_name, filepath))
 
         write_init(skill_name, skill_path, level_name)
-        return
     else:
-        # Populate file with BasicLevel
-        with open(os.path.join(skill_path, "__init__.py"), 'r') as fp:
+        # Add level to skills/<new_skill>/__init__.py
+        filepath = os.path.join(skill_path, "__init__.py")
+        with open(filepath, 'r') as fp:
             filedata = fp.read()
 
         replace = ",\n        BasicLevel('{level_name}', pkg_resources.resource_filename(__name__, '_{level_name}/'))\n    ]".format(level_name=level_name)
         filedata = filedata.replace("\n    ]", replace)
         filedata = filedata.replace("[,", "[")
-        filepath = os.path.join(skill_path, "__init__.py")
+
         with open(filepath, 'w') as fp:
             fp.write(filedata)
             print("Registered level \"{}\" in {}".format(level_name, filepath))
-    return
 
 
 def write_test(skill_name, skill_path, level_name, level_path):
```

Comment:
`skill_name` is not used, you can remove it as an argument

Reply:
done